frontend/appflowy_flutter/lib/plugins/ai_chat/presentation/message/ai_message_action_bar.dart

```diff
@@ -184,6 +185,19 @@ class CopyButton extends StatelessWidget {
       ),
     );
   }
+
+  String _getTrimmedPlainText(String plainText) {
+    // match and capture inner url as group
+    final singleLineMarkdownImageRegex =
```

Comment:
You can move this regex to `common_patterns.dart` file.

Reply:
Done!